hack/build-rpms.sh

```diff
@@ -31,7 +31,7 @@ tar czf "${rpm_tmp_dir}/SOURCES/${OS_RPM_NAME}-test.tar.gz" \
 	.
 cp -r "${ci_data}/." "${rpm_tmp_dir}/SOURCES"
 
-echo "${OS_GIT_COMMIT}"
+yum-config-manager --disable extras-source
```

Comment:
@umohnani8 Could you delete this line and add this instead: `sed -i 's/vault.centos.org\/centos\/$releasever/vault.centos.org\/centos\/7.7.1908/g' /etc/yum.repos.d/CentOS-Sources.repo` . This will ensure the URL is correct is fetched.

Reply:
Done, thanks!